gemini-resin-archetype/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/configuration/App-Dev.conf

```diff
@@ -16,6 +16,7 @@ Extends = App-Base.conf
 DeploymentDescription = Development/${Servlet.MachineName}
 
 # Database connectivity for Development.
+# TODO: EDIT THIS
```

Comment:
The meta here is hard to follow - is this a `TODO` for you on this PR, or for the developer who builds a Gemini project from this archetype?

Reply:
The Gemini configuration files already had a number of "TODO" comments intended for the developer building a Gemini project from the archetype. I just added some more.